echemdb/data/cv/entry.py

```diff
@@ -100,6 +100,38 @@ def __getitem__(self, name):
         """
         return Descriptor(self.package.descriptor)[name]
 
+    def citation(self, render='text'):
+        r"""
+        Returns a formatted reference for the entry's bibliography either as:
+    
+        J. Doe, et al., Journal Name, volume (YEAR) page, "Title"
+
+        Rendering default is plain text 'text', but can be changed to any format 
+        supported by pybtex, such as markdown 'md', 'latex' or 'html'. 
```

Comment:
I would prefer to add a link to the `pybtex` documentation, but I did not find anything useful in the API documentation.
So probably we leave it to the user to find its way through `pybtex` code.